client/components/chart/index.jsx

```diff
@@ -148,8 +166,8 @@ function Chart( {
 				) }
 			</div>
 			{ ! isPlaceholder && (
-				<div className="chart__y-axis">
-					<div className="chart__y-axis-width-fix">{ numberFormat( 100000 ) }</div>
+				<div ref={ yAxisRef } className="chart__y-axis">
+					<div className="chart__y-axis-width-fix">{ numberFormat( 1e5 ) }</div>
```

Comment:
Is the scientific notation necessary for the fix, or just a choice? If not, it adds a tiny bit of friction when reading, but no biggie – was just curious.

Reply:
Not necessary, just a choice. In my opinion, I find scientific notation easier to parse than counting the number of zeros. Especially if the number of zeros > 3!